src/Core/Style.js

```diff
@@ -130,6 +130,18 @@ const textAnchorPosition = {
     'top-left': [0, 0],
 };
 
+/**
+ * Define Getter/Setter for a specifique parameter of one of the five Style categories (fill, stroke, point, text, icon)
+ * and zoom of a Style instance.
+ * When the getter of a categorie.parameter is called, the purpose is to send, in that order, the value set by the user 'userValue'
+ * (if available), or the value read from the data 'dataValue' or the default value 'defaultValue' (if set). In case we
+ * need to overpass all we can set a value using the setter.
```

Comment:
I propose to update the comment a bit:

```
Defines a new style property and its getter and setter for a given parameter in a given category (one of fill, stroke, point, text, icon or zoom).
The getter is in charge of returning the right style value from the following ones if they are defined (in that specific order):
* the value set by the user (`userValue`)
* the value read from the data source (`dataValue`)
* the default fallback value (`defaultValue`)
```

Reply:
Thanks for that proposal that is indeed clearer that mine.